src/gui/inputdevices/AbstractInputHandler.cpp

```diff
@@ -5,6 +5,7 @@
 #include "AbstractInputHandler.h"
 #include "InputContext.h"
 #include <gui/XournalppCursor.h>
```

Comment:
replace <> with "" for local headers if you need to touch them.
Write a empty line between all sorts of different libraries.
Write a line between the local headers and the other xournalpp headers.

Reply:
no need to change those though